mirgecom/diffusion.py

```diff
@@ -74,21 +118,40 @@ def grad_facial_flux_central(kappa_tpair, u_tpair, normal):
 def grad_facial_flux_weighted(kappa_tpair, u_tpair, normal):
     r"""Compute the numerical flux for $\nabla u$.
 
-    Weights each side's value by the corresponding thermal conductivity $\kappa$:
+    Weights each side's value by the corresponding thermal conductivity $\kappa$,
+    for an isotropic material, given by
+
+    .. math::
+
+        F = -\frac{\kappa^- u^- + \kappa^+ u^+}{\kappa^- + \kappa^+} \hat{n},
+
+    or an orthotropic material, as
 
     .. math::
 
-        F = -\frac{\kappa^- u^- + \kappa^+ u^+}{\kappa^- + \kappa^+} \hat{n}.
+        F = -\frac{(\kappa^- \cdot \hat{n}) u^- + (\kappa^+ \cdot \hat{n}) u^+}
+            {\kappa^- \cdot \hat{n} + \kappa^+ \cdot \hat{n}} \hat{n}
```

Comment:
I don't know if we have solid justification that it should look this way (though I also don't have any better suggestions). It would be good to check this with a verification case.

Reply:
Matt, I tested having "multiple temperatures" (https://github.com/illinois-ceesd/mirgecom/blob/5a0ea41b6924acb7ba9dfb9b9c9957fc2e7cf48c/mirgecom/diffusion.py#L149-L150) or the product with the normal for a simple diffusion case (see `heat-diffusion.py`).

In practice, the results are the same (see below). I added a comment in the code to keep this in mind so we don't forget (https://github.com/illinois-ceesd/mirgecom/pull/976/commits/3f1dda6c9f90e5c6fa467698f95e24c7fe0addb6).

dx, dy, error, p

multiple temperature
0.03278688524590164 0.03225806451612903 0.027683064208580785 2
0.01639344262295082 0.016129032258064516 0.0030159150665184957 2

single temperature (dot normal)
0.03278688524590164 0.03225806451612903 0.02768306420858256 2
0.01639344262295082 0.016129032258064516 0.003015915066520936 2